pydicom/pixel_data_handlers/util.py

```diff
@@ -574,14 +574,21 @@ def apply_windowing(
 
 
 def convert_color_space(
-    arr: "np.ndarray", current: str, desired: str
+    arr: "np.ndarray",
+    current: str,
+    desired: str,
+    per_frame: Optional[bool] = False
```

Comment:
Shouldn't this be just `bool` instead of `Optional[bool]`?

Reply:
Whoops, fixed